src/module_7/database/mongoDB.ts

```diff
@@ -0,0 +1,48 @@
+import mongoose, { ConnectOptions } from "mongoose"
+import * as UserEntities from "./schemas/user.entity"
+import * as ProductEntities from "./schemas/product.entity"
+import User from "./documents/user.document"
+import Product from "./documents/product.document"
+
+const startMongo = () => {
+  const uri: string = "mongodb://localhost:27017"
+  const options: ConnectOptions = {
+    dbName: "mydatabase",
+    user: "root",
+    pass: "nodegmp",
```

Comment:
It is a bad practice to use hard coded credentials, instead, try using dotenv and have .env file to store the credentials like environment variables.